README.md

```diff
@@ -112,6 +117,11 @@ If the `subnet_id` refers to a subnet configured to disallow public IPs on any a
     oci_profile_name: "DEFAULT"
     ssh_keypath: "~/.ssh/id_rsa.pub"
     post_create_script: >-
+    preemptible_instance: true
```

Comment:
I think some people might blindly cut and paste this example and get a preemptible instance without realizing it.  You could leave this in, but perhaps have it commented out with a quick explanation?

Reply:
That's fair.  i guess the same goes for the `shape_config` as well.  My whole goal of including it here was to show what it would look like rather than provide a copy/paste example, but I see your point.